chapters/ko/_toctree.yml

```diff
@@ -0,0 +1,173 @@
+- title: 0. Setup
+  sections:
+  - local: chapter0/1
+    title: Introduction
+
+- title: 1. 트랜스포머 모델
+  sections:
+  - local: chapter1/1
+    title: 단원 소개
+  - local: chapter1/2
```

Comment:
We only need the sections of the table of contents for content that is already translated. In this case, you can remove everything from this section onwards :)

Reply:
I just fix all the things you mentioned above & completed chapter0 and 1!
Please check out!